Blackfynn/sparc-app
# Upgrade Nuxt to 2.11.0 from 2.0.0

# Description

The purpose of this PR is to upgrade Nuxt to 2.11.0 from 2.0.0. The upgrade process was pretty seamless, and all functionality remains.

## Type of change


# How Has This Been Tested?

To fully test out this change, you should start from scratch. This remains removing any installed modules and start fresh.
- Remove `/node_modules` with `rm -rf /node_modules`
- Install node modules with `yarn`
- Run the app with `yarn dev`
- App should function normally

# Checklist:
